feg/radius/src/modules/coafixedip/coa_fixed_ip_test.go

```diff
@@ -80,6 +79,16 @@ func TestCoaFixed(t *testing.T) {
 	require.Equal(t, res.Code, radius.CodeDisconnectACK)
 }
 
+func waitForRadiusServerToBeReady(secret []byte, port int) {
+	MaxRetries := 10
```

Comment:
This function should not exist three times. Maybe you can move it to where `"fbc/cwf/radius/modules"` is, since this is already imported in all the coa test files.

Reply:
It is now moved to a central place at `fbc/cwf/radius/modules/modules_test_utils.go`, where several tests in the folder will call it from.